docs/2.0/docs/accountfactory/architecture/security-controls.md

```diff
@@ -0,0 +1,231 @@
+# Controls
+
+Gruntwork Account Factory employs a defense-in-depth approach to secure workflows across both GitHub and GitLab platforms. This document outlines the controls Pipelines uses to ensure that only infrastructure written in code and approved by a reviewer can be deployed to your AWS accounts.
+
+Account Factory relies on Pipelines to drive infrastructure changes via GitOps workflows, so make sure to read the [Pipelines security controls](/2.0/docs/pipelines/architecture/security-controls) for more details on how Pipelines secures workflows.
+
+## Least privilege principle
+
+Pipelines adheres to the principle of least privilege, granting only the necessary permissions for infrastructure actions.
+
+By default, the only repository/group required to interact with infrastructure using Pipelines in Gruntwork Account Factory is the `infrastructure-live-root` repository/group. This contains infrastructure code for `management`, `logs`, `security`, and `shared` accounts. Access should be limited to a small, trusted group responsible for defining critical infrastructure, similar to the role of the `root` user in Unix systems.
+
+## Platform-Specific Access Controls
+
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+
+<Tabs groupId="platform">
+<TabItem value="github" label="GitHub" default>
+
+- The AWS IAM role assumed via OIDC when pull requests are opened or updated has a trust policy that restricts access to the repository itself and provides read-only permissions
+- The AWS IAM role assumed via OIDC when pull requests are merged into the `main` branch has a trust policy limiting access to the repository's `main` branch and granting write permissions
+- Branch protection rules can be configured to require reviews and status checks
+- GitHub App or machine user authentication options available
+
+</TabItem>
+<TabItem value="gitlab" label="GitLab">
+
+- The AWS IAM role assumed via OIDC when merge requests are opened or updated has a trust policy that restricts access to the group itself and provides read-only permissions
+- The AWS IAM role assumed via OIDC when merge requests are merged into the `main` branch has a trust policy limiting access to the group's `main` branch and granting write permissions
+- Protected branches can be configured to require approvals and pipeline success
+- Machine user authentication required with group-level access configuration
+
+</TabItem>
+</Tabs>
+
+## Infrastructure access control
+
+An optional `infrastructure-live-access-control` repository/group can manage access control for infrastructure provisioned in AWS accounts. Using this is a best practice for centralized and auditable access management.
```

Comment:
Here we describe this repo as optional but in the repo-topology doc, we assert this as the canonical way Account Factory works. Update to make consistent: Either the repo-topology asserts some repos as optional, or this doesn't specify the access-control repo as optional.